### PR TITLE
expand iptables command to be more readable

### PR DESCRIPTION
**Pull request by rickypai**
use verbose flags and organize into different lines to make the command more readable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,8 +121,26 @@ probably want to restrict proxy access to the docker0 interface!
...
+  --dport 80 \
+  --in-interface docker0 \
+  --jump DNAT \
+  --protocol tcp \
+  --table nat \
+  --to-destination $LOCAL_IPV4:8000 \
+  --wait
+
+/sbin/iptables
+  --wait80 \
```
Comment (ryan-lane) [inline README.md]: `--wait`, rather than `--wait80`?

**Comment by rickypai** [comment]
fixed. seems like i copypasta something wrong in the github edit UI :/

File: README.md (hunk 1 above)
Comment (rickypai) [inline README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -112,8 +121,26 @@ probably want to restrict proxy access to the docker0 interface!
...
+  --insert INPUT 1
+  --protocol tcp \
+  --dport
```
Comment (rickypai) [inline README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -112,8 +121,26 @@ probably want to restrict proxy access to the docker0 interface!
...
+  --table nat \
+  --to-destination $LOCAL_IPV4:8000 \
+  --wait
+
+/sbin/iptables
+  --wait80 \
+  --insert INPUT 1
+  --protocol tcp \
+  --dport 
+  ! \
```
Comment (rickypai) [inline README.md]: fixed

**Comment by ryan-lane** [comment]
Looks good. Thanks for the commit!

**Pull request merged**